test/distributed/test_nccl.py

```diff
@@ -19,12 +21,14 @@
 
 class TestNCCL(TestCase):
 
+    @unittest.skipIf(TEST_WITH_ROCM, 'Skip NCCL tests for ROCm')
```

Comment:
nit: link the issue in the string

Reply:
Which issue should I link to here?

Reply:
You should file an issue that these tests are failing on ROCm, label it with "topic: flaky-tests", and then link that issue.

Reply:
(Make sure to copy the logs demonstrating these tests are failing on ROCm into the issue you create.)

Reply:
(see https://fb.quip.com/24NyAwEsCd9a#QEJACADUdxe)